src/migrations/20251740066033_user_info.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  await knex.schema.createTable('user_info', table => {
+    table.increments();
+    table.string('sub', 50).notNullable().comment('Subject - Identifier for the End-User at the Issuer.');
+    table.string('name', 200).nullable().comment('End-User\'s full name in displayable form including all name parts, possibly including titles and suffixes, ordered according to the End-User\'s locale and preferences.');
+    table.string('middle_name').nullable().comment('Casual name of the End-User that may or may not be the same as the given_name.');
```

Comment:
This comment mixes up `nick_name`, middle name is just the second name.

Reply:
Oooh great catch!